js/repl/compile.js

```diff
@@ -58,6 +58,7 @@ export default function compile(code: string, config: CompileConfig): Return {
   let useBuiltIns = false;
   let spec = false;
   let loose = false;
+  let corejs = "2";
```

Comment:
```suggestion
  let corejs = "3";
```
nit: let's make the defaults be modern